strawberry/extensions/tracing/apollo.py

```diff
@@ -130,6 +120,14 @@ def get_results(self):
         return {"tracing": self.stats.to_json()}
 
     async def resolve(self, _next, root, info, *args, **kwargs):
+        if should_skip_tracing(_next, info):
+            result = _next(root, info, *args, **kwargs)
+
+            if isawaitable(result):
+                result = await result
```

Comment:
I think we add `# pragma: no cover` here, I don't think we can make a test for this since no default resolver or introsspection one are async. :)

Reply:
We are still missing something. 😕

Reply:
I'll push a test for this case:

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/667029/95109606-b9183f80-0734-11eb-9a15-dae2c3562155.png">